### ISSUE TITLE
Accessibility: Orphaned or Missing Form labels

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#5]
WCAG 2.0- 1.1.1 Non-text Content Level (A) - violation 'All form inputs have associated text labels.

1.Go to: data.ottawa.ca
2.Observe the below image of identified wave tool 'red' image identifying either;
A. Missing form label missing - does not have a corresponding form label.
B. Orphaned form label - a form label is present but not associated with the form.

Resolution: Add text labels to all form inputs. Ensure all orphaned form labels are associated with the form.

![tab trap within tables not accessible](https://cloud.githubusercontent.com/assets/5271253/8012479/e3eb5964-0b8f-11e5-8a2a-03705e44ddf2.JPG)
